packages/devtools_app/lib/src/screens/app_size/app_size_table.dart

```diff
@@ -126,8 +126,8 @@ class _SizeColumn extends ColumnData<TreemapNode> {
 
   @override
   int compare(TreemapNode a, TreemapNode b) {
-    final Comparable valueA = getValue(a);
-    final Comparable valueB = getValue(b);
+    final Comparable valueA = getValue(a) as Comparable;
+    final Comparable valueB = getValue(b) as Comparable;
```

Comment:
are these casts still necessary after changing the return type of getValue?

Reply:
Unfortunately yes. They were hidden by using `dynamic`.